buildroot-external/package/hardkernel-boot/hardkernel-boot.mk

```diff
@@ -32,6 +32,63 @@ define HARDKERNEL_BOOT_BUILD_CMDS
 	dd if=$(@D)/u-boot.img of=$(@D)/u-boot.gxbb bs=512 skip=96
 endef
 
+else ifeq ($(BR2_PACKAGE_HARDKERNEL_BOOT_ODROID_C4),y)
+# travis/odroidc4-153 20201014
+HARDKERNEL_BOOT_VERSION = 2a4e31e3fde9bced8a150d338aa397a0362df191
+
+HARDKERNEL_BOOT_BINS += u-boot.sm1
+define HARDKERNEL_BOOT_BUILD_CMDS
+	# blx_fix_g12a.sh is appropriate for C4 as well as N2
+	curl -L -o $(@D)/fip/blx_fix.sh https://raw.githubusercontent.com/home-assistant/hassos-blobs/d88ec078470f236d694ba1f9e69a44759ddf41ea/odroid-n2/blx_fix_g12a.sh
+  # get these bits from Neil Armstrong's repo; probably will want to host these
+  # in hassos-blobs (or maybe they're the same as the G12B versions?)
+	curl -L -o $(@D)/fip/acs.bin https://gitlab.com/superna9999/amlogic-boot-fip/-/raw/master/odroid-c4/acs.bin
+	curl -L -o $(@D)/fip/bl301.bin https://gitlab.com/superna9999/amlogic-boot-fip/-/raw/master/odroid-c4/bl301.bin
+
+	bash $(@D)/fip/blx_fix.sh \
+		$(@D)/fip/g12a/bl30.bin $(@D)/fip/zero_tmp $(@D)/fip/bl30_zero.bin \
+		$(@D)/fip/bl301.bin $(@D)/fip/bl301_zero.bin $(@D)/fip/bl30_new.bin \
```

Comment:
Make a PR to home-assistant/hassos-blobs for this files

Reply:
Created PR https://github.com/home-assistant/hassos-blobs/pull/1.